corehq/apps/app_manager/static/app_manager/js/app-notifications.js

```diff
@@ -0,0 +1,24 @@
+/* globals hqDefine moment */
+hqDefine('app_manager/js/app-notifications.js', function () {
+    function NotifyFunction(userId, $element) {
+        return function(msg) {
+            var msgObj = JSON.parse(msg);
+            // only show notifcations from other users
+            if (msgObj.user_id !== userId) {
+                var $inner;
+                // if nothing in the outer element, add the notification div and populate
```

Comment:
I looked at this and the code is way simpler, but the UI feels a bit clunkier (to me). Not sure whether you think better to do it this way? 

![image](https://cloud.githubusercontent.com/assets/66555/19009174/b8617c26-8779-11e6-941d-2544c695e564.png)